pkg/ibmvpc/explainability_test.go

```diff
@@ -867,3 +883,17 @@ func TestMultiExplainabilityOutput(t *testing.T) {
 		"connection vsi to vsi")
 	fmt.Println("\n\n", outputString)
 }
+
+func TestInputLBPrivateIP(t *testing.T) {
+	vpcConfigMultiVpc := getConfig(t, "iks_config_object")
+	require.NotNil(t, vpcConfigMultiVpc, "vpcConfigMultiVpc equals nil")
+
+	pipCidr := "192.168.36.6"
+	cidr2 := "192.168.4.4"
+	// should fail since pip address can not be an explainability input
+	_, err1 := vpcConfigMultiVpc.ExplainConnectivity(pipCidr, cidr2, nil)
+	fmt.Println(err1.Error())
+	require.NotNil(t, err1, "the test should fail since "+pipCidr+" is a Private IP address")
+	require.Equal(t, "illegal src: no network interfaces are connected to "+pipCidr, err1.Error())
```

Comment:
Is it suffice to provide the "usual" error message here? Would it help to mention this is a potential pip?

Reply:
also the addresses of non potential private ip are not allowed to be inputs